memegen/settings.py

```diff
@@ -28,6 +30,8 @@ class TestConfig(Config):
     DEBUG = True
     TESTING = True
 
+    SQLALCHEMY_DATABASE_URI = "postgresql://localhost/memegen_test"
```

Comment:
[`testing.postgresql`](https://pypi.python.org/pypi/testing.postgresql/) is good for creating test databases for tests since it requires no manual setup, but unless you need stuff like `HSTORE` or full text search, you can just use a sqlite database.